README.md

```diff
@@ -2,9 +2,16 @@
 
 [![Build Status](https://travis-ci.com/pseewald/fprettify.svg?branch=master)](https://travis-ci.com/pseewald/fprettify) [![Coverage Status](https://coveralls.io/repos/github/pseewald/fprettify/badge.svg?branch=master)](https://coveralls.io/github/pseewald/fprettify?branch=master) [![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0) [![PyPI version](https://badge.fury.io/py/fprettify.svg)](https://badge.fury.io/py/fprettify)
 
-fprettify is an auto-formatter for modern Fortran code that imposes strict whitespace formatting, written in Python.
+This is a fork of [pseewald/fprettify](https://github.com/pseewald/fprettify),
```

Comment:
@zaikunzhang you probably didn't mean to include changes to README in this PR.

Reply:
Oh, I'm could really use the `--keep-blank-lines` option, for that I contribute to some projects where (many) consecutive blank lines between subroutines are pervasively used (and expected in contributions, at least preserved).

@zaikunzhang, thank you very much for the proposal, can you remove the committed edit to the README, so that the PR could be merged (and hopefully the PyPi version bumped)?